pulumi/src/docker.ts

```diff
@@ -59,7 +59,7 @@ export const buildAndPushImage = async (args: BuildAndPushImageArgs): Promise<vo
 
   dockerBuildArgs.push(args.context)
 
-  execSync(`${envs} docker build ${dockerBuildArgs.join(' ')}`, { stdio: 'inherit' })
+  execSync(`${envs} docker build --platform linux/amd64 ${dockerBuildArgs.join(' ')}`, { stdio: 'inherit' })
```

Comment:
want to hit all of the other `docker run` or `docker build` commands throughout with the same treatment?

Reply:
Ah yeah forgot there was more